### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by privatebin-translator**

### CODE REVIEW

**Comment by jaideejung007** [comment]
Hello, I'm the one translating your Crowdin Thai project; your work is excellent. I hope you'll merge my Thai translation in your master branch; I'm looking forward to it. 🙂

**Review by elrido — APPROVED** [review]

**Comment by elrido** [comment]
@jaideejung007 for the follow up work in #1015 I need one of the other maintainers to review it, before it can get merged (we set up the project so all changes require a second pair of eyes).

**Pull request merged**
